index.js

```diff
@@ -0,0 +1,7 @@
+const include = __dirname;
+const isNodeApiBuiltin = process.versions.modules >= 52;
```

Comment:
For versions of node without N-API built-in, the N-API version will be the same as the node-api package version. For versions of node with N-API built-in, the N-API version will be the same as the node version. It's tricky because the former is determined at build time, while the latter is determined at run-time. This is an area we know we need to further define and document the behavior and best practices for native module developers.